js/body_mixin.ts

```diff
@@ -0,0 +1,230 @@
+/** @module fly
```

Comment:
We do not use this annotation in other modules.  It adds no value to the intellisense for TypeScript, so I wouldn't recommend using it.  Also what is `fly` anyways?

Reply:
Why is this module named `body_mixin`?  It is not a mixin in any sense.  It exports a class named `DenoBody`, which implements `domTypes.Body`.  It should just be named `body.ts`

Reply:
Okay. I'll fix that.